src/seedu/addressbook/AddressBook.java

```diff
@@ -87,12 +87,10 @@
     private static final String MESSAGE_USING_DEFAULT_FILE = "Using default storage file : " + DEFAULT_STORAGE_FILEPATH;
 
     // These are the prefix strings to define the data type of a command parameter
-    private static final String PERSON_DATA_PREFIX_PHONE = "p/";
-    private static final String PERSON_DATA_PREFIX_EMAIL = "e/";
+    protected static final String PERSON_DATA_PREFIX_PHONE = "p/";
```

Comment:
"protected" was used to limit the access level as much as possible, although it may not be necessary in this scenario.